### DISCUSSION TITLE
endpoint for getting friends

### DISCUSSION THREAD

**Discussion by krupupakku** [RetroAchievements/RAWeb#2346] [Ideas]
Hey all.

I would like to propose the creation of a new API endpoint for retrieving the friends list. 
Currently, the information available at https://retroachievements.org/friends.php can only be accessed using the doRequest.php endpoint, which we are trying to avoid.  

Thank you for considering this proposal.

**Comment by gastongaiduk** [comment]
https://github.com/RetroAchievements/RAWeb/pull/2887
